lesson-33/hw_33.js

```diff
@@ -0,0 +1,75 @@
+
+const subjects = {
+    mathematics: {
+        students: 200,
+        teachers: 6
+    },
+    biology: {
+        students: 120,
+        teachers: 6
+    },
+    geography: {
+        students: 60,
+        teachers: 2
+    },
+    chemistry: {
+        students: 100,
+        teachers: 3
+    }
+}
+
+const subjectsKeys = Object.keys(subjects);
+const subjectsValues = Object.values(subjects);
+
+// task 1 
+
+const nameArray = subjectsKeys.join(', ');
+console.log(nameArray);
+
+// task 2
+
+const manCount = subjectsValues.reduce((accum, current, iterator) => {
+    if (iterator === 1) {
+        const firstSubjectPeopleArray = Object.values(accum);
+        accum = {students: 0, teachers: 0}
+        accum.students += firstSubjectPeopleArray[0];
+        accum.teachers += firstSubjectPeopleArray[1];
+    }
+    const subjectPeopleArray = Object.values(current);
+    accum.students += subjectPeopleArray[0];
+    accum.teachers += subjectPeopleArray[1];
+    return accum;
+});
```

Comment:
in `reduce `method you can pass accumulating value as second argument after callback function 
```})
}, { students: 0, teachers: 0 });
```

Reply:
I got it, I'll fix it